src/invidious/playlists.cr

```diff
@@ -437,17 +437,26 @@ def get_playlist_videos(db, playlist, offset, locale = nil, continuation = nil)
     db.query_all("SELECT * FROM playlist_videos WHERE plid = $1 ORDER BY array_position($2, index) LIMIT 100 OFFSET $3",
       playlist.id, playlist.index, offset, as: PlaylistVideo)
   else
-    if offset >= 100
-      # Normalize offset to match youtube's behavior (100 videos chunck per request)
-      offset = (offset / 100).to_i64 * 100_i64
+    if continuation
+      initial_data = YoutubeAPI.next({
+        "videoId"    => continuation,
```

Comment:
Please use `video_id` instead of `continuation` (I know that it was the original term used, but that should be fixed). Please also propagate accordingly, but keep the `&continuation=` API parameter as it is (we will do what's required to deprecate this).

Reply:
Gladly added it. `continuation` sounds weird to me